examples/tgi-llama-inference-aws-inferentia/requirements.txt

```diff
@@ -0,0 +1 @@
+runhouse[sky]
```

Comment:
`runhouse[aws]` instead? i'm trying to fix this so we don't need to re-run `pip install "skypilot[aws]"` after the install